Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).


It also includes following components:

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/keycloak/admin-ui/horizontal-auto.svg)

jonkoops commented on themes/src/main/resources-community/theme/base/account/messages/messages_de.properties:
```diff
@@ -339,3 +339,46 @@ access-denied-when-idp-auth=Zugriff verweigert bei Authentifizierung mit {0}
+locale_ja=日本語
+locale_it=Italiano
+error-user-attribute-required=Bitte geben Sie Attribut {0} an.
+locale_ru=Русский
+locale_uk=Українська
+locale_zh-CN=中文简体
+locale_tr=Türkçe
+locale_fi=Suomi
```
Hmmm, I thought we were going to get rid of this, should we hold off on this? @ahus1

ahus1 replied:
The locales appeared in to message resources, and I set them to empty and not translated only in one of them. I fixed it now in the Weblate UI.

ahus1 commented on themes/src/main/resources-community/theme/base/account/messages/messages_de.properties:
```diff
@@ -339,3 +338,21 @@ access-denied-when-idp-auth=Zugriff verweigert bei Authentifizierung mit {0}
+error-pattern-no-match=Ungültiger Wert.
+error-invalid-uri-scheme=Ungültiges URL-Schema.
+error-invalid-date=Ungültiges Datum.
+error-user-attribute-read-only=Das Feld {0} ist read only.
```
Could you please try to find a translation for "read only" as well?

robson90 replied:
Then "schreibgeschützt"? Or something like "Auf das Feld {0} haben Sie nur lesenden Zugriff"

ahus1 replied:
I'd think I'd go for "schreibgeschützt".

robson90 replied:
Fixed.

ahus1 reviewed (changes requested):
Thank you @robson90 for the translations. See one comment below where it needs IMHO a refinement.

ahus1 reviewed (dismissed):
Thank you for these translations!

robson90 committed:
Translated using Weblate (German)

Translation: Keycloak/Admin UI
Translate-URL: https://hosted.weblate.org/projects/keycloak/admin-ui/de/

Translated using Weblate (German)

Translation: Keycloak/Theme base/account
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseaccount/de/

Translated using Weblate (German)

Translation: Keycloak/Theme base/account
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseaccount/de/

Co-authored-by: Robin <39960884+robson90@users.noreply.github.com>
Signed-off-by: Robin <39960884+robson90@users.noreply.github.com>

ahus1 committed:
Translated using Weblate (German)

Translation: Keycloak/Account UI
Translate-URL: https://hosted.weblate.org/projects/keycloak/account-ui/de/

Translated using Weblate (German)

Translation: Keycloak/Theme base/account
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseaccount/de/

Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>

weblate committed:
Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translation: Keycloak/Theme base/account
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseaccount/

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
